src/store/Configuration/mutations.ts

```diff
@@ -0,0 +1,38 @@
+import { ConfigurationStateIF } from '@/interfaces'
+import Vue from 'vue'
+import axios from '@/axios-auth'
+
+export default {
+  setConfiguration (state: ConfigurationStateIF, data: any) {
+    state.configuration = data
+  },
+  /** Use this mutator to set a specific attribute in unit testing.
+   * Like this: store.commit('setTestConfiguration', ['AUTH_WEB_URL', 'https://auth.web.url/'])
+   * */
+  setTestConfiguration (state: ConfigurationStateIF, payload: any) {
+    if (!state.configuration) {
+      Vue.set(state, 'configuration', {})
+    }
+    Vue.set(state.configuration, payload[0], payload[1])
+  },
+  setSessionVariables (state: ConfigurationStateIF, responseData: any) {
```

Comment:
How about a blank line between functions?

Reply:
That's a good idea.  It's not how the /store/mutations.js is formatted but I'm happy to set that as a pattern going forward.